src/libraries/System.Text.Json/tests/Serialization/TestClasses.GenericCollections.cs

```diff
@@ -861,6 +861,83 @@ IEnumerator IEnumerable.GetEnumerator()
         }
     }
 
+    public class GenericIDictionaryWrapperPrivateConstructor<TKey, TValue> : IDictionary<TKey, TValue>
```

Comment:
We can just derive `GenericIDictionaryWrapper` here to save some LOC:

```C#
public class GenericIDictionaryWrapperPrivateConstructor<TKey, TValue> : GenericIDictionaryWrapper<TKey, TValue>
{
    private GenericIDictionaryWrapperPrivateConstructor() { }
}
```

Reply:
fixed in c070a9c and af2633a